hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/query/matcher/GraterThanMatcher.java

```diff
@@ -0,0 +1,41 @@
+package pl.allegro.tech.hermes.management.infrastructure.query.matcher;
+
+import org.apache.commons.jxpath.JXPathException;
+import pl.allegro.tech.hermes.management.infrastructure.query.graph.ObjectGraph;
+
+public class GraterThanMatcher implements Matcher {
+
+    private final String attribute;
+
+    private final Object rightSideValue;
+
+    public GraterThanMatcher(String attribute, Object rightSideValue) {
+        this.attribute = attribute;
+        this.rightSideValue = rightSideValue;
+    }
+
+    @Override
+    public boolean match(Object value) {
+        try {
```

Comment:
this try-catch is a bit wide, maybe we should divide it into smaller parts (separated graph navigation, greater-than decision logic)?